Add scenario for provider use tutorial (#182)
hashicorp/katakoda: terraform-provider-intro/read-coffee-ingredients.md

> Add the following to your `main.tf`{{open}} file. This will retrieve ingredients for the first coffee from your order. The `data block` retrieves additional information about a resource, which enables it to be referenced by another Terraform resource. In this example, it's used by an `output` block to display the coffee ingredients.

Reviewer: ```suggestion
Add a data block to retrieve ingredients for the first coffee from your order. A data block retrieves additional information about a resource, which allows you to reference it in another Terraform resource. In this example, you will use it in an `output` block to display the coffee ingredients.
 
 Add the following to your `main.tf`{{open}} file.
```
